pkgs/applications/video/glaxnimate/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, zlib
+, potrace
+, ffmpeg
+, libarchive
+, python3
+, qtbase
+, qttools
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "glaxnimate";
+  version = "0.5.1";
+
+  src = fetchFromGitLab {
+    owner = "mattbas";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "G4ykcOvXXnVIQZUYpRIrALtDSsGqxMvDtcmobjjtlKw=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+  buildInputs = [
+    zlib
+    potrace
+    ffmpeg
+    # gst_all_1.gst-libav # Note that since gst-libav-1.6, libav is actually ffmpeg
+    libarchive
+    python3
+    qtbase
+    qttools
+  ];
+
+  meta = with lib; {
+    homepage = "https://gitlab.com/mattbas/glaxnimate";
+    description = "Simple vector animation program.";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ tobiasBora ];
+    platforms = platforms.linux;
```

Comment:
Sure it doesn't build on darwin? There are installation instructions for macOS: https://gitlab.com/mattbas/glaxnimate#macos Maybe it works with some simple darwin-specific CMAKE flags?

Reply:
I've no ideas, but as I can't test on darwin I don't know what I should put. Maybe someone from @NixOS/darwin-maintainers can test? Btw @turion do you want to be part of tge maintainers as you certainly use this software more than me?

Reply:
You could add a simple integration test, then ofborg can test it.

I wouldn't want to be maintainer. I use kdenlive, but have never used glaxnimate, nor do I intend to.

Reply:
I am building it on x86-64_darwin

Reply:
seem fine 
<img width="491" alt="image" src="https://user-images.githubusercontent.com/29592775/211196989-eb008200-0617-4343-acf4-afbc677c892d.png">

Reply:
Does it compile on windows ? if no add `platforms = platforms.unix;` (try to cross compile with mingw)

Reply:
I guess the MinGW cross chain is mostly broken though...